packages/ui-core/src/components/toggle/toggle.component.styles.ts

```diff
@@ -0,0 +1,93 @@
+import styled, { css } from 'styled-components';
+import { colors } from '@keen.io/colors';
+import { MIN_TOGGLE_WIDTH } from './toggle.component';
```

Comment:
There is a circular dependency here - between `toggle.component` and `toggle.component.styles`.
Please - move **constant** values to separate file.